stubs/Pillow/PIL/FpxImagePlugin.pyi

```diff
@@ -13,4 +14,9 @@ class FpxImageFile(ImageFile):
     format: ClassVar[Literal["FPX"]]
     format_description: ClassVar[str]
     fp: _OleStream | None
+    maxid: int
+    rawmode: str
+    jpeg: dict[int, Incomplete]
+    tile_prefix: Incomplete
```

Comment:
Where's this one defined?

Reply:
https://github.com/python-pillow/Pillow/blob/main/src/PIL/FpxImagePlugin.py#L221

Reply:
Weird, wonder how I missed that. Thanks!